src/functions/envstore.ts

```diff
@@ -0,0 +1,96 @@
+import { firebaseApiOrigin } from "../api";
+import { FirebaseError } from "../error";
+import { Client } from "../apiv2";
+
+interface EnvStoreEntry {
+  name?: string;
+  vars: Record<string, string>;
+}
+
+const apiClient = new Client({ urlPrefix: firebaseApiOrigin, auth: true });
+
+/**
+ * Get envstore entry from the EnvStore Service.
+ *
+ * @return {EnvStoreEntry} EnvStore entry for given envstore id.
+ */
+export async function getStore(projectId: string, envStoreId: string): Promise<EnvStoreEntry> {
+  let response;
+  try {
+    response = await apiClient.get<EnvStoreEntry>(
+      `/v1alpha/projects/${projectId}/envStores/${envStoreId}`
+    );
+  } catch (err) {
+    throw new FirebaseError(`Failed to make request: ${err.message}`, { original: err });
```

Comment:
Here and everywhere else, you'll probably thank yourself in the future if your debug log lines describe the request that failed.